.env.example

```
@@ -7,12 +7,11 @@ CLIENT_SECRET=MCP_password_here
-SLACK_SIGNING_SECRET=your_slack_signing_secret_here
-SLACK_WEBHOOK_URL=your_slack_webhook_url_here
+SLACK_APP_TOKEN=your_slack_app_level_token_here
 
 # Port for the application to listen on (default: 3001)
 PORT=3001
 
 # Security: API key for test endpoints (e.g., /test/refresh)
 # Generate a secure random string for production use
-TEST_API_KEY=your_secure_test_api_key_here 
\ No newline at end of file
+TEST_API_KEY=your_secure_test_api_key_here 
```

dotenv-linter (4.0.0)

[warning] [TrailingWhitespace] Trailing whitespace detected

(TrailingWhitespace)